apps/datahub/README.md

```diff
@@ -1,147 +1,7 @@
 # Datahub
 
-UI of the web application `datahub`.
+The Datahub is an application to provide a default, pure and simple UI for metadata and dataset search.
 
-`datahub` is an application to provide a default, pure and simple UI for metadata and dataset search.
+More information here: https://geonetwork.github.io/geonetwork-ui/main/docs/apps/datahub.html
 
-Inspire by Opendata catalogs (CKAN, Opendatasoft), the Hub will host geo and non-geo dataset. It will provide dataviz tool and focuses the experience on the dataset instead of on the metadata.
-
-## How to run it
-
-The Datahub application is available as a docker image or as a ZIP archive.
-
-### With docker
-
-The docker image is `geonetwork/geonetwork-ui-datahub`.
-
-To run it on the 8080 port with a custom GN4 API url and proxy path, use:
-
-```bash
-$ docker run -p 8080:80 \
-             -e GN4_API_URL=https://gn4.custom/geonetwork/srv/api \
-             -e PROXY_PATH=/proxy?url= \
-             geonetwork/geonetwork-ui-datahub
-```
-
-Notice how the `GN4_API_URL` and `PROXY_PATH` variables are used to override any values present in the app configuration file.
-**This override will happen everytime the docker container is started.**
-
-The application will be available on http://localhost:8080/datahub/.
-
-#### Specifying a custom configuration file
-
-If the `GN4_API_URL` and `PROXY_PATH` environment variables are not enough and you want to specify a full configuration file,
-you can do so like this:
```

Comment:
Maybe it's worth mentioning that this is currently still only available/used in the datahub (if I remember right)?

Reply:
it's actually available for any app using the default Dockerfile :)